about.html

```diff
@@ -0,0 +1,117 @@
+                <div class="menu-icon">
+                    <img src="./img/menu-icon.png" alt="Menu Icon">
+                    <ul class="nav-menu">
+                        <span class="times">X</span>
+                        <li class="mobile-menu">
+                            <a href="./index.html">Home</a>
+                        </li>
+                        <li class="mobile-menu">
+                            <a href="#ourspeakers">Speakers</a>
+                        </li>
+                        <li class="mobile-menu">
+                            <a href="#sponsor">Sponsors</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="menu-desktop">
+                    <div class="menu-atlas">
+                        <img src="./img/logo.png" alt="Menu Icon">
+
+                    </div>
+                    <ul class="menu-ul">
+                        <li class="menu-li">
+                            <a href="./index.html">Home</a>
+                        </li>
+                        <li class="menu-li">Tickets</li>
+                        <li class="menu-li">Join</li>
+                        <li class="menu-li">Sponsor</li>
+                        <li class="menu-li">News</li>
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="about-section">
+                <div class="intro">
+                    <h1 class="about-h1"> "CREATIVE COMMONS <br>GLOBAL SUMMIT ,<br> 2015"</h1>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet voluptatum vero velit, ad
+                        incidunt ex, eligendi aliquid suscipit doloremque culpa nihil tempore placeat nostrum veniam
+                        similique aliquam rerum nisi neque atque. Unde quas iure et id illo necessitatibus debitis fuga
+                        expedita illum impedit voluptate veniam in molestias itaque tenetur quia ipsam, dignissimos
+                        ipsum, esse ratione eveniet obcaecati. A nemo ducimus mollitia tenetur officiis eveniet, dolores
+                        unde asperiores iste optio accusamus possimus laudantium qui! Nihil saepe totam numquam magnam
+                        aperiam quasi.</p>
+
+                    <h6>Please Contack us if you need any furthur information about this Bootcamp</h6>
+
+
+                </div>
+            </div>
+            <div class="about-our-logo">
+                <h2> CREATIVE COMMONS GLOBAL SUMMIT 2015 LOGO</h2>
+                <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus animi veritatis rem nesciunt
+                    temporibus, adipisci expedita voluptatibus culpa illo saepe et nisi suscipit quidem voluptatem!
+                    Tempora aliquam rem necessitatibus facilis.</p>
```

Comment:
Great job so far 👍.
[OPTIONAL] I think you can centralize the `p` tag to match what we have in the design and also to make the project looks better.
| Figma Design |Your Design |
| :---:   | :---: | 
| <img width="638" alt="image" src="https://user-images.githubusercontent.com/73854999/203860556-cb0f94f0-9be3-46b0-867f-e203cf909fdb.png">| <img width="1388" alt="image" src="https://user-images.githubusercontent.com/73854999/203860515-83070f3d-94da-4baa-9170-c6c036fe093b.png">|